XenAdmin/Dialogs/NetworkingProperties.cs

```diff
@@ -496,6 +497,13 @@ private void AcceptBtn_Click(object sender, EventArgs e)
             // list originally so that they can be used as a filter against downPIFs.
             newPIFs.RemoveAll(delegate(PIF p) { return !p.Changed; });
 
+            newNamePIFs.RemoveAll(delegate(PIF p) { return !p.Changed; });

```

Comment:
Let's modernise a bit this code and the 2 lines above it by using lambdas.

Reply:
will fix it.